<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UC_Alerts</class>
 <widget class="QFrame" name="UC_Alerts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="Frame_Alerts">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>860</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Frame_Alerts_Rents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1031</width>
       <height>795</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>رقم العقد</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>إسم المالك</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>رقم هاتف المالك</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>إسم الميتأجر</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>رقم هاتف المستأجر</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الباقى من المده</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>تاريخ الإنتهاء</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QFrame" name="Frame_Alerts_Offers">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget_5">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>1028</width>
       <height>795</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <column>
      <property name="text">
       <string>رقم العقار</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>إسم المالك</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>رقم هاتف المالك</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>المتبقى من المده</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>تاريخ الانتهاء</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QFrame" name="Frame_Alerts_Requests">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1028</width>
       <height>795</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>رقم الطلب</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>إسم العميل</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>رقم الهاتف</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>نوع العقار المطلوب</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_alert_offers_frame">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>إشعارات العروض</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_alert_rent_frame">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>إشعارات الإيجارات</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_alert_requests_frame">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>إشعارات الطلبات</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
